### PR TITLE
Added a section explaining why a fork was created.

### PR DESCRIPTION
**Pull request by almet**
I was wondering why the fork was created, so dropped over IRC had approximately
this answer, that I rephrased and put in a pull request.

I'm actually not sure about the part where I say decisions are taken
collegially, but I would love to see this a reality :-)

Please, don't hesitate to add some more information there in the comments,
I will integrate them in the pull request and hopefully everything will be
merged to master at some point.

### CODE REVIEW

**Comment by MaxLeiter** [comment]
But of a nitpick, but maybe “Why the fork?” sounds better?

**Comment by almet** [comment]
Thanks, I updated accordingly.

File: README.md [hunk 1]
```diff
@@ -20,6 +20,19 @@ __What features does it have?__
...
 - _.. and more!_
 
+__Why the fork?__
+
+We felt that the original [Shout](https://github.com/erming/shout) project
+"stagnated" a little because its original author wanted it to remain his pet
+project (which is a perfectly fine thing!).
+
+A bunch of people, excited about making things a bit differently than the upstream
+project forked it under a new name: «The Lounge».
```
Comment (astorije) [inline README.md]: s/«The Lounge»/"The Lounge"/ :-)

File: README.md [hunk 2]
```diff
@@ -20,6 +20,19 @@ __What features does it have?__
...
+
+This forks aims to be community managed, meaning that the decisions are taken
```
Comment (astorije) [inline README.md]: "This **fork**"

**Comment by astorije** [comment]
@almet, thanks for this!
On top of the 2 comments I left you inline, could you squash your commits please? I should be :+1: then, but the second reviewer for this should be a native speaker ideally.

File: README.md [hunk 3]
```diff
@@ -20,6 +20,19 @@ __What features does it have?__
...
+in a collegial fashion, and that a bunch of maintainers should be able to make
+the review process more streamlined.
```
Comment (dgw) [inline README.md]: "more streamlined—and ideally quicker."

Thoughts, @astorije?

File: README.md [hunk 4]
```diff
@@ -20,6 +20,19 @@ __What features does it have?__
 - Responsive layout — works well on your smartphone
 - _.. and more!_
 
+__Why the fork?__
+
+We felt that the original [Shout](https://github.com/erming/shout) project
+"stagnated" a little because its original author wanted it to remain his pet
+project (which is a perfectly fine thing!).
+
+A bunch of people, excited about making things a bit differently than the upstream
```
Comment (dgw) [inline README.md]: Either "doing things" or "a bit different" would sound better, I think, but there's nothing technically wrong with the existing phrasing.

**Comment by dgw** [comment]
I'm not technically a reviewer, but I am a native English speaker. :smirk_cat:

Left a couple inline comments of my own. If I *was* a reviewer, I'd give this a :+1: after @astorije's comments are addressed.

File: README.md (hunk 3 above)
Comment (astorije) [inline README.md]: I would not use words like "should" or "ideally" here actually, but more like "goal(s)", "intent", more present tense. But maybe that doesn't make any sense and therefore you should ignore this comment :-)

File: README.md (hunk 4 above)
Comment (astorije) [inline README.md]: "doing things" :+1:

File: README.md (hunk 3 above)
Comment (dgw) [inline README.md]: Maybe then, "the review process quicker and more streamlined."

Comment (astorije) [inline README.md]: :+1:

**Comment by almet** [comment]
updated and squashed.

**Comment by MaxLeiter** [comment]
> A bunch of people, excited about doing things a bit differently than the upstream
 project forked it under a new name: “The Lounge”
A bunch of people, excited about doing things a bit differently than the upstream
 project, forked Shout under a new name: “The Lounge”


>This fork aims to be community managed
The fork aims to be community managed

**Comment by astorije** [comment]
Thanks for your thoughts @MaxLeiter but I believe this is good as is :-)

@almet, :+1:, thanks for this!

**Comment by xPaw** [comment]
:+1:

**Pull request merged**